src/MembershipsSDKAdapter.js

```diff
@@ -238,13 +268,18 @@ export default class MembershipsSDKAdapter extends MembershipsAdapter {
 
       // Emit on membership updates
       meeting.members.on('members:update', (payload) => {
+        logger.debug(...LOG_ARGS, 'getMeetingMembers()', ['received members: update event', {payload}]);
```

Comment:
'received "members:update" event', can be without the double quotes but no space after `:`